packages/adp-tooling/src/preview/adp-preview.ts

```diff
@@ -91,7 +116,15 @@ export class AdpPreview {
         private readonly config: AdpPreviewConfig,
         private readonly project: ReaderCollection,
         private readonly logger: ToolsLogger
-    ) {}
+    ) {
+        this.cache = new NodeCache();
+        this.routesHandler = new RoutesHandler(project, logger, this.cache);
+        this.rateLimiter = rateLimit({
```

Comment:
While this is very useful for a productive server, this middleware runs only in a dev environment, so you can remove the rate limiter code, and I will mark the corresponding sonar error as false positive.

Reply:
Removed in https://github.com/SAP/open-ux-tools/pull/1183/commits/14175ce266a6695f01a40102dbf1a99c50c0b4e4
and in https://github.com/SAP/open-ux-tools/pull/1183/commits/b8b42a09ac4bf1a7e85e93d8903657b47187eb24